src/parser/parse_tree_cl.c

```diff
@@ -9164,6 +9171,18 @@ pt_print_spec (PARSER_CONTEXT * parser, PT_NODE * p)
       q = pt_append_nulstring (parser, q, "(");
       r1 = pt_print_bytes_l (parser, p->info.spec.entity_name);
       q = pt_append_varchar (parser, q, r1);
+
+      if (p->info.spec.remote_server_name)
```

Comment:
This code only runs in the following cases.
But the result is different from what I expected.

If such syntax should be allowed, it is considered a bug.

**On the remote server**
```
csql> select * from (t1, t2);

           c1
=============
          200
          100
```

**Use DBLink**
```
csql> select * from (t1, t2);

           c1
=============
          200
          100

csql> select * from t1@cubrid_dblink;

           c1
=============
          100

csql> select * from t2@cubrid_dblink;

           c1
=============
          200

csql> select * from (t1@cubrid_dblink, t2@cubrid_dblink);

There are no results.
0 row selected. 

csql> select * from dblink (cubrid_dblink, 'select c1 from (t1, t2)') as t1 (c1 int);

           c1
=============
          200
          100
```

Reply:
The blow case also returns error:

select * from (t1, t2);

In line 1, column 22,

ERROR: before ' ); '
There are no attributes in ([dba.t1], [dba.t2]).

Reply:
![image](https://user-images.githubusercontent.com/68979537/220535318-11872fb5-f8d5-46eb-ad3b-7c184c497d8f.png)

It runs just fine...

Reply:
The query from (table, table) is not support yet as remote query.